app/src/main/res/values/translatable.xml

```diff
@@ -258,6 +258,9 @@ THE SOFTWARE.
 	<string name="coverloader_shadow_title">Load artwork from hidden folder</string>
 	<string name="coverloader_shadow_summary">Try to load artwork from \'/sdcard/Music/.vanilla/ARTIST/ALBUM.jpg\'</string>
 
+	<string name="coverloader_inline_title">Load artwork from ID3 tag</string>
+	<string name="coverloader_inline_summary">Try to load a cover stored in files ID3 tag</string>
```

Comment:
I assume that this would also work for other supported containers, such as M4A and Ogg?

Reply:
translation made more generic